core/banking-protocol/src/test/resources/docker-compose.yml

```diff
@@ -16,11 +16,11 @@ services:
       - SPRING_DATASOURCE_USERNAME=postgres
       - SPRING_DATASOURCE_PASSWORD=docker
       - PKCS12_KEYSTORE=/pkcs12/sample-qwac.keystore
-    build: /Users/maxim/IdeaProjects/open-banking-gateway/core/banking-protocol
+    build: /Users/maxim/IdeaProjects/open-banking-gateway/opba-embedded-starter
```

Comment:
Why commiting your local file path, it can be relative or passed through env. variable

Reply:
fixed